src/view/components/microbit/MicrobitImage.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import "../../styles/Microbit.css";
+import { MICROBIT_SVG } from "./Microbit_svg";
+
+interface EventTriggers {
+    onMouseUp: (button: HTMLElement, event: Event) => void;
+    onMouseDown: (button: HTMLElement, event: Event) => void;
+    onMouseLeave: (button: HTMLElement, event: Event) => void;
+}
+interface IProps {
+    eventTriggers: EventTriggers;
+}
+
+// Displays the SVG and call necessary svg modification.
+export class MicrobitImage extends React.Component<IProps, any> {
+    componentDidMount() {
+        const svgElement = window.document.getElementById("microbit_svg");

```

Comment:
Look at how refs and findDOMNode works, there are always preferable ways than getElementById

Reply:
I will be using a reference as suggested, it is best practice in case of duplicates